cypress/e2e/collection.spec.js

```diff
@@ -0,0 +1,12 @@
+// Test for clicking collection item from home page 
+
+describe('Clicking into a collection ', () => {
+it('finds the content "type"', () => {
+    cy.visit("http://localhost:3000");
```

Comment:
```suggestion
    cy.visit("/");
```
Here as well

Reply:
Actually just confirmed with mentors, we should test in localhost and not on the actual website. So ignore this. 

One suggestion is we could change the base URL in the cypress config file so we don't have to write the full path every time.